src/es/taurusfansub/src/eu/kanade/tachiyomi/extension/es/taurusfansub/TaurusFansub.kt

```diff
@@ -2,17 +2,16 @@ package eu.kanade.tachiyomi.extension.es.taurusfansub
 
 import eu.kanade.tachiyomi.multisrc.madara.Madara
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
-import eu.kanade.tachiyomi.source.model.SManga
 import org.jsoup.nodes.Document
 import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
 class TaurusFansub : Madara(
     "Taurus Fansub",
-    "https://taurus.topmanhuas.org",
+    "https://lectortaurus.com",
     "es",
-    dateFormat = SimpleDateFormat("dd/MM/yyy", Locale.ROOT),
+    dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.ROOT),
```

Comment:
FYI, this is not a typo and [actually allowed](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html) according to the docs.

> For parsing, if the number of pattern letters is more than 2, the year is interpreted literally, regardless of the number of digits.

You don't need to change anything, just resolve on ACK.